### DISCUSSION TITLE
Logs or some verbose mode?

### DISCUSSION THREAD

**Discussion by Gounlaf** [getseabird/seabird#47] [General]
:wave: 

I'm trying to load a kubeconfig and seabird freeze totaly. Where can I find logs? or is there a verbose mode?
It's installed via flatpak on Ubuntu 22.04.

Before opening any issue, I would like at least to find a possible cause.

Thanks.

Regards.

**Comment by jgillich** [comment]
You can run it in the terminal to get logs: `flatpak run dev.skynomads.Seabird`

**Reply by dbirks** [reply]
Hm I'm troubleshooting a freezing issue as well, and don't get any output from that command:

![image](https://github.com/getseabird/seabird/assets/7545665/59e240d9-fb94-4076-b70c-489c150a719f)

**Reply by jgillich** [reply]
When does that happen, when you click load or when you select the file? Need more info here. The icons are broken so something isn't right

**Reply by Gounlaf** [reply]
Hi @jgillich,

1. New cluster :heavy_check_mark: 
2. Load kubeconfig :heavy_check_mark: 
3. Select file from file selector :heavy_check_mark: 
4. Choose on of the available context :heavy_check_mark: 
5. Click on confirm :x: the freeze happens here

I will try to update my kubeconfig progressively in complexity.

I have multiple servers, context, and users (some with token, some with authentication plugin)

**Reply by jgillich** [reply]
Ah, I see the problem

**Reply by Gounlaf** [reply]
Just tested with a simple kubeconfig where there is a minikube only (auth based on client certifcate + key), and it works like a charm :)

**Reply by jgillich** [reply]
Fixed in 7fda146d415ada570be67b444a7bba6bce032859

**Reply by Gounlaf** [reply]
Thanks!
